controllers/user_api_tokens.go

```diff
@@ -79,6 +80,11 @@ func (c *ApiController) AddApiToken() {
 		return
 	}
 
+	token := tokenUser.AccessKey + tokenUser.AccessSecret
+	isRootUser := currentUser.Owner == "built-in" && currentUser.Name == "admin"
```

Comment:
Please add constants for "built-in" and "admin" in this context, or a helper function

Reply:
done

Reply:
There's one more case in this PR where it is not yet done

Reply:
sorry, I missed this one, now it's done)